### ISSUE TITLE
Summarize your Suggestion Here

### ISSUE THREAD

**Issue by Arivelo1** [surfbryce/beautiful-lyrics#773]
### What are you Suggesting?

a Feature

### What is the context behind this Suggestion?

I prefer to not have my music auto-scroll as with fast paced music it is hard to read/sing

When listening to a song it is hard to follow when the lyrics skip with no fluidity. Fluidity from an animation would help but I would like to be able to turn on a manual scroll on songs that have sync.

### What is your Suggestion?

Fluidity from an animation would help but I would like to be able to turn on a manual scroll on songs that have sync.
- Easier to sing along/follow
- Able to read ahead without being skipped around

### Suggestion Checklist

- [X] You clearly titled/explained your Suggestion.
- [X] You accurately described/contextualized the reasoning behind your Suggestion.
- [X] IF applicable to your Suggestion, you provided visual examples/references to accurately represent what you are Suggesting.

**Comment by surfbryce** [comment]
@Arivelo1 I would rewrite this suggestion request with examples of what you are talking about. Also you are able to manually scroll and it will stop auto-scrolling until the active lyrics comes back into view.
